src/item.ts

```diff
@@ -66,7 +66,12 @@ export interface ItemProps {
   caption?: string
 
   /**
-   * Html content, if you need to use it as modal
+   * Slide content
+   */
+  content?: JSX.Element
+
+  /**
+   * Slide html content
    */
   html?: string
```

Comment:
Is it still actual for React wrapper to have html prop if we can provide JSX content?

Although, it still can be useful for some people/cases 🤔

Reply:
I think we will leave it at this stage